libs/community/langchain_community/vectorstores/meilisearch.py

```diff
@@ -99,24 +91,20 @@ def __init__(
         self._embedding = embedding
         self._text_key = text_key
         self._metadata_key = metadata_key
-        self._embedders = embedders
-        self._embedders_settings = self._client.index(
-            str(self._index_name)
-        ).update_embedders(embedders)
 
     def add_texts(
         self,
         texts: Iterable[str],
         metadatas: Optional[List[dict]] = None,
         ids: Optional[List[str]] = None,
-        embedder_name: Optional[str] = "default",
+        embedder: str = "default",
         **kwargs: Any,
     ) -> List[str]:
         """Run more texts through the embedding and add them to the vector store.
 
         Args:
             texts (Iterable[str]): Iterable of strings/text to add to the vectorstore.
-            embedder_name: Name of the embedder. Defaults to "default".
+            embedder: Name of the embedder. Defaults to "default".
```

Comment:
Should we keep a default value here or remove it and make the parameter mandatory, to be more in line with Meilisearch?